app/main.py

```diff
@@ -0,0 +1,39 @@
+import os
+from fastapi import FastAPI, HTTPException, status
+from fastapi.middleware.cors import CORSMiddleware
+from llama_cpp import Llama
+import dotenv
+
+dotenv.load_dotenv()
+
+llama = Llama(
+    model_path=os.getenv('LLM_MODEL_PATH'),
+    n_gpu_layers=-1,
+    # seed=1337,
+    n_ctx=2048,
+)
+
+app = FastAPI()
+
+
+app.add_middleware(
```

Comment:
Is this to allow it to get prompts from the front end? what is the format of the prompts, is the context and then the data all sent in the query string at once or does it remember past prompts?

Reply:
Also for the sake of reference the data coming in would be from https://github.com/2023-gpig-a/DMAS/blob/main/util/data_generators.py